doc/tools/kconfig/functions.rst

```diff
@@ -0,0 +1,28 @@
+.. _functions:
+
+Kconfig Functions
+#################
+
+The Kconfiglib provides a means for user defined preprocessor functions.  We
+utilize the functionality in Zephyr to expose Device Tree information to
+Kconfig.  This allows us the ability to have Kconfig extract default values
+for Kconfig symbols from device tree or for us to depend on Device Tree data.
+
+Device Tree Related Functions
+*****************************
+
+dt_get_int_val:
```

Comment:
Should you add the parameters to these functions?  Something like:
```
dt_get_int_val (name, unit)
```

Reply:
Added